elastic/kibana
# [reporting] show loading state when creating a reporting job

<img width="200" alt="Screen Shot 2023-04-13 at 12 04 26 PM" src="https://user-images.githubusercontent.com/373691/231845867-303fe34d-5032-49cf-8408-dc7b7725e6e8.png">


### Steps to test
* Load your favorite sample data set and open its dashboard
* Click "Share" and then click "PDF Reports"
* Open browser devtools and open network tab. Turn on network throttling to better see loading state
* Click "Generate PDF". Notice how button now gives feedback its clicked and something is happening. Before, button would not show loading state and users are confused into thinking nothing is happening.
